packages/@n8n/plugin-sdk/src/backend/types.ts

```diff
@@ -0,0 +1,7 @@
+export type BackEndModuleContext = {};
+
+export type BackEndModuleSetupFn = (context: BackEndModule) => void;
+
+export type BackEndModule = {
```

Comment:
Here and elsewhere, can we avoid the middle cap on both FE and BE? Easier to read, less prone to typos, and seems the official spelling:

https://en.wikipedia.org/wiki/Frontend_and_backend

Reply:
Good point. Updated